include/SFML/System/Vector2.inl

```diff
@@ -25,7 +25,7 @@
 
 ////////////////////////////////////////////////////////////
 template <typename T>
-inline Vector2<T>::Vector2() :
+constexpr inline Vector2<T>::Vector2() :
```

Comment:
`constexpr` alone is enough, see https://stackoverflow.com/q/14391272
Also other occurrences.

Reply:
The same is true of `template`: https://stackoverflow.com/a/10535711/598696

Since `inline` was there already, I assumed that we wanted to be more explicit (yet redundant) about it. Do you still want me to remove all `inline` specifiers?

Reply:
True, good point. Not sure why it was originally there.

In my opinion, `inline` can be removed, yes. Let's see what others think.

Reply:
@eXpl0it3r, @binary1248: any opinion on the matter?

Reply:
I think anybody slightly more experienced with C++ would know that templates are implicitly inline, thus the `inline` keyword would just be noise for them (for the compiler it is basically useless since it does not change anything). `constexpr` on the other hand does modify compilation and thus is not only an indicator to the user but the compiler as well.

I don't think we should base API design around the idea of educating users on the principles of C++. If someone wants to know whether a templated function is inline they should be reading a good C++ book instead of coming to us. 😉